Mage/src/main/java/mage/abilities/effects/common/DrawCardSourceControllerEffect.java

```diff
@@ -16,24 +15,38 @@
  */
 public class DrawCardSourceControllerEffect extends OneShotEffect {
 
+    public enum DrawCardsPhrasing {
```

Comment:
I don't like this being nested here - seems like there should be one public enum for all usages